src/lib/docs/SessionKit/LoginResult.md

```diff
@@ -1,19 +1,18 @@
 ---
 title: LoginResult
-description: The data returned from a call to the login method from the Session Kit
-category: SessionKit
+description: The data returned from a call to the login method from the Session Kit.
+category: Session Kit
```

Comment:
We cannot have spaces in these `category` names, it breaks the URL generation for some reason.